docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -311,27 +328,28 @@ JKS truststores are single files containing multiple certificates.
 You can use the alias to select the appropriate certificate when multiple certificates are present.
 However, avoid using the JKS format as it is less secure than PKCS12.
 
-To configure a JKS truststore, use the following properties:
-
+* To configure a JKS truststore:
++
 [source,properties]
 ----
 quarkus.tls.trust-store.jks.path=client-truststore.jks
 quarkus.tls.trust-store.jks.password=password
 quarkus.tls.trust-store.jks.alias=my-alias
 ----
-
-`.jks` files are password-protected, so you need to provide the password to open the truststore. 
++
+`.jks` files are password-protected, so you need to provide the password to open the truststore.
 However, unlike keystores, the alias does not require a password because it contains a public certificate, not a private key.
 
+
 ==== Credential providers
```

Comment:
There seems to be repetition of the same-named sections (but with different content) - PEM, JKS, Credentials providers...

I don't think that's a good thing for a reference doc. Perhaps try to think about if the same-named sections could be joined together, somehow...